### PR TITLE
[Workers & Hyperdrive] Update Workers PostgreSQL docs to include Postgres.js and Hyperdrive

### PR DESCRIPTION
**Pull request by thomasgauvin**

### CODE REVIEW

File: src/content/docs/workers/tutorials/postgres/index.mdx [hunk 1]
```diff
@@ -285,6 +295,62 @@ You can now use your Cloudflare Worker to insert new rows into the `products` ta
...
+# Pasted from the output of `wrangler hyperdrive create <NAME_OF_HYPERDRIVE_CONFIG> --connection-string=[...]` above.
+[[hyperdrive]]
+binding = "HYPERDRIVE"
+id = "<ID OF THE CREATED HYPERDRIVE CONFIGURATION>"
+```
+
+Create the types for your Hyperdrive binding using the following command:
+
+```bash
+npm run cf-typegen
```
Comment (irvinebroque) [inline src/content/docs/workers/tutorials/postgres/index.mdx]: Should this be `npx wrangler types`?

**Review by kodster28 — APPROVED** [review]

File: src/content/docs/workers/tutorials/postgres/index.mdx (hunk 1 above)
Comment (thomasgauvin) [inline src/content/docs/workers/tutorials/postgres/index.mdx]:
```suggestion
npx wrangler types
```

Comment (thomasgauvin) [reply src/content/docs/workers/tutorials/postgres/index.mdx]: done!

File: src/content/docs/workers/tutorials/postgres/index.mdx [hunk 2]
```diff
@@ -285,6 +295,62 @@ You can now use your Cloudflare Worker to insert new rows into the `products` ta
...
 
+## 8. Use Hyperdrive to accelerate queries
+
+Create a Hyperdrive configuration using the connection string for your PostgreSQL database.
+
+```bash
+npx wrangler hyperdrive create <NAME_OF_HYPERDRIVE_CONFIG> --connection-string="postgres://user:password@HOSTNAME_OR_IP_ADDRESS:PORT/database_name"
+```
+
+You can also use explicit parameters by following the [wrangler documentation for Hyperdrive](https://developers.cloudflare.com/workers/wrangler/commands/#create-1).
```
Comment (thomasgauvin) [inline src/content/docs/workers/tutorials/postgres/index.mdx]:
```suggestion
You can also use explicit parameters by following the [wrangler documentation for Hyperdrive](/workers/wrangler/commands/#create-1).
```

**Pull request merged**